public/rooms.json

```diff
@@ -0,0 +1,56 @@
+[
```

Comment:
I like the forethought going into extensibility, specifically with "Name" and "Image". However, I think "floor" might be redundant if you can just scrape the left-most digit of the room number.

I get that this might not generalize to buildings with basements or irregular room numbers, but with our only events in the future being in Klaus and IC, I think the floor tag is redundant

Also, since all this file does right now is map number to floor, and if we're getting rid of the floor in favor of programmatically obtaining it, we can probably get rid of this file